content/go/concepts/Variables/Variables.md

```diff
@@ -0,0 +1,124 @@
+
+---
+Title: 'Variables'
+Description: 'Variables are used to store numbers, strings, true/false boolean states, or objects based on their type when declaring them.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Variables'
+  - 'Booleans'
+  - 'Strings'
+  - 'Integers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+## Definition
+
+Variables are used to store data such as numbers, strings, or Booleans in memory for later use.
+
+## Difference From Constants 
+
+Unlike constants, when declaring a variable, its value can be reassigned later on but with the restriction that the new value's data type is the same as the data type when initialized.
```

Comment:
```suggestion
## Difference From Constants 

Unlike constants, the value of variables can be reassigned after declaration, but its data type must remain the same as it was when first initialized. For example, an `int` variable can start as 2 and be reassigned as 0 (another `int`), but it can't be reassigned as "Codecademy" (a `string`).
```